core/services/flux_monitor.go

```diff
@@ -236,6 +243,51 @@ func (f pollingDeviationCheckerFactory) New(initr models.Initiator, runManager R
 	)
 }
 
+// ExtractFeedURLs extracts a list of url.URLs from the feeds parameter of the initiator params
+func ExtractFeedURLs(feeds models.Feeds, orm *orm.ORM) ([]*url.URL, error) {
+	var feedsData []interface{}
+	var urls []*url.URL
+
+	err := json.Unmarshal(feeds.Bytes(), &feedsData)
+
+	if err != nil {
+		return nil, err
+	}
+
+	for _, entry := range feedsData {
+		var bridgeURL *url.URL
+		var err error
+
+		switch feed := entry.(type) {
+		case string: // feed url - ex: "http://example.com"
+			bridgeURL, err = url.ParseRequestURI(feed)
+		case map[string]interface{}: // named feed - ex: {"bridge": "bridgeName"}
```

Comment:
Can you match on `map[string]string` here?

Reply:
I tried - doesn't look like it.

Reply:
Thanks for taking a look at that.